Documentation/ApiOverview/Bootstrapping/Index.rst

```diff
@@ -7,60 +7,79 @@ Bootstrapping
 =============
 
 TYPO3 CMS has a clean bootstrapping process driven mostly
-by class :php:`\TYPO3\CMS\Core\Core\Bootstrap`. This class
-contains a host of methods each responsible for a little
-step along the initialization of a full TYPO3 process,
-be it the backend or other contexts.
+by class :php:`\TYPO3\CMS\Core\Core\Bootstrap`. This class is initialized by
+calling  :php:`Bootstrap::init()` and serves as an entry point for later calling
+an Application class, depending on the TYPO3_MODE and several context-dependant
```

Comment:
I think we should keep regular terms like `application` in lowercase unless we refer to a specific "thing" which will then very likely be a class name.

Also we should not refer to `TYPO3_MODE` anymore which is deprecated in TYPO3v11.